update examples/ directory (#146)
C2Devel/terraform-provider-croccloud: examples/elb/README.md

# ELB Example

> The ELB (Elastic Load Balancing) example launches the nginx web server and creates an ELB with necessary network resources.

Reviewer: может дописать типа `launches the nginx web server on target instance...`, а то еще подумают что елб это nginx

Author: окей, поправлю

# ELB Example

> The ELB example launches the nginx web server and creates an ELB with necessary network resources.

Reviewer 2: хотя бы в первом вхождении надо в скобках расшифровать, что такое ELB
Elastic Load Balancer же?)

Author: Почти, Elastic Load Balancing

Докинула расшифровку, для EKS тоже

> This example assumes an SSH key pair was created in the cloud console.

Reviewer 2: ```suggestion
This example assumes that an SSH key pair was created in the cloud console.
```

Running the example:

```
$ export AWS_ACCESS_KEY_ID="your-access-key"
$ export AWS_SECRET_ACCESS_KEY="your-secret-key"
$ terrafom init
$ terraform apply -var="key_name=your-key-name"
```

> In a few minutes the nginx welcome page will be available at the ELB DNS name.

Reviewer 2: ```suggestion
In a few minutes, you can reach the nginx welcome page at the ELB DNS name.
```
